repondeur/zam_repondeur/views/import_json.py

```diff
@@ -53,7 +53,7 @@ def import_backup(context: LectureResource, request: Request) -> Response:
         elif counter["articles"]:
             message = f"{counter['articles']} article(s) chargé(s) avec succès"
         request.session.flash(Message(cls="success", text=message))
-        lecture.modified_at = datetime.utcnow()
```

Comment:
We don’t need the modified_at any more?

Reply:
AFAIK, [you removed it](https://github.com/betagouv/zam/pull/403) ;)

It was re-introduced here when I [restored the feature](https://github.com/betagouv/zam/pull/414) and given that if doesn't do any harm it stayed there silently since then.

Reply:
Ok so let’s remove it a second time then!